src/data/faq_de.json

```diff
@@ -127,6 +127,17 @@
                 "title": "EU Zertifikate",
                 "id": "eu_dcc",
                 "accordion": [
+                    {
+                    "title": "Welche EU-Zertifikate werden unterstützt?",
+                    "anchor": "eu_dcc_which",
+                    "active": true,
+                    "textblock": [
+                        "Ab <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> der Corona-Warn-App (CWA) werden alle drei Arten der EU digitalen COVID-Zertifikate unterstützt:",
+                        "<ul>  <li>digitales Impfzertifikat</li>  <li>digitales Genesenenzertifikat</li>  <li>digitales Testzertifikat</li> </ul>",
+                        "Bisher wurden digitale Impfzertifikate erstmalig mit <a href='../../de/blog/2021-06-10-cwa-version-2-3' target='_blank' rel='noopener'>Version 2.3</a> unterstützt, gefolgt von der zusätzlichen Unterstützung von digitalen Testzertifikate in <a href='../../de/blog/2021-06-24-cwa-version-2-4' target='_blank' rel='noopener'>Version 2.4.</a> Mit der Veröffentlichung der <a href='../../de/blog/2021-07-12-cwa-version-2-5' target='_blank' rel='noopener'>Version 2.5</a> wurden digitale Genesenenzertifikate zusätzlich unterstützt.",

```

Comment:
Typo: ... von digitalen Testzertifikate**n** in ...

Reply:
@DerVogel2020 
Thanks for spotting the typo. I will submit a new PR to correct it.